.pre-commit-config.yaml

```diff
@@ -22,7 +22,7 @@ repos:
     rev: 19.10b0
     hooks:
       - id: black
-        language_version: python3.6
+        language_version: python3
```

Comment:
Remove this

Reply:
We need a PR to update this.

Reply:
yup, it accidentally got dragged in with the rest of my modifications.